cms/djangoapps/contentstore/rest_api/v1/serializers.py

```diff
@@ -21,6 +21,21 @@ class LimitedProctoredExamSettingsSerializer(serializers.Serializer):
     enable_proctored_exams = serializers.BooleanField()
     proctoring_provider = serializers.CharField()
     proctoring_escalation_email = serializers.CharField(allow_blank=True)
+    create_zendesk_tickets = serializers.BooleanField(required=False)
```

Comment:
This field is set to not be required right now so that it can be compatible with the current POST requests being sent . Once https://github.com/edx/frontend-app-course-authoring/pull/111 has been merged and deployed, the field can be required.